### PR TITLE
Layout - dropdown searchable, layout required

### PR DESCRIPTION
**Pull request by ainouzgali**
…able from layouts list

### What change does this PR introduce?
Screen Shot 2023-01-19 at 13 27 26
Screen Shot 2023-01-19 at 13 27 41

### Why was this change needed?

Also a small fix - preheder vars were not rendered to var manager

### CODE REVIEW

File: apps/web/src/components/templates/email-editor/EmailInboxContent.tsx [hunk 1]
```diff
@@ -44,11 +47,12 @@ export const EmailInboxContent = ({
             <Controller
               name={`steps.${index}.template.subject` as any}
               control={control}
-              render={({ field, fieldState }) => {
+              render={({ field }) => {
                 return (
                   <Input
                     {...field}
-                    error={fieldState.error?.message}
+                    required
```
Comment (p-fernandez) [inline apps/web/src/components/templates/email-editor/EmailInboxContent.tsx:54]: Do we need to set all the input fields as required?

Comment (ainouzgali) [reply apps/web/src/components/templates/email-editor/EmailInboxContent.tsx:54]: No, only subject and now layout are required in email editor.  wdyt @p-fernandez ?

Comment (p-fernandez) [reply apps/web/src/components/templates/email-editor/EmailInboxContent.tsx:54]: I thought this was rendering multiple inputs for different fields. Misread.

**Review by p-fernandez — APPROVED** [review]
Let's review the existing tests and merge it. 👍🏻

**Pull request merged**